recipes/poolsnp/build.sh

```diff
@@ -0,0 +1,33 @@
+#! /bin/bash
+
+set -ex
+
+BASE_DIR=$(dirname "$0")
```

Comment:
`BASE_DIR` is the same as the already provided `RECIPE_DIR`

Reply:
I needed the path for the work directory where the github files are downloaded to set the relative paths for the `scripts/` and the `TestData/` directories. 

`RECIPE_DIR` returns the root of the recipes directory, whereas `BASE_DIR` here returns the root of the work directory for `conda-bld`

```
'BASE_DIR is set to: $HOME/miniconda3/conda-bld/poolsnp_1721646158691/work'
'RECIPE_DIR is set to: $HOME/bioconda-recipes/recipes/poolsnp'
```